__tests__/app.test.js

```diff
@@ -52,6 +52,172 @@ describe("/api/articles", () => {
         });
       });
   });
+  describe("/api/articles/(queries)", () => {
+    describe("?topic", () => {
+      test("GET: 200 - querying an existing topic responds with articles on that topic", () => {
+        return request(app)
+          .get("/api/articles/?topic=mitch")
+          .expect(200)
+          .then(({ body: { articles } }) => {
+            expect(articles.length).toBeGreaterThanOrEqual(1);
+            articles.forEach((article) => {
+              expect(article).toMatchObject({
+                topic: "mitch",
+              });
+            });
+          });
+      });
+      test("GET: 200 - when no topic query provided responds with all articles", () => {
+        return request(app)
+          .get("/api/articles/?topic")
+          .expect(200)
+          .then(({ body: { articles } }) => {
+            expect(articles.length).toBe(testData.articleData.length);
+            articles.forEach((article) => {
+              expect(article).toMatchObject({
+                topic: expect.any(String),
+              });
+            });
+          });
```

Comment:
We don't really want this behaviour, if a query is malformed it should throw an error.